R/rhelpers.R

```diff
@@ -0,0 +1,263 @@
+#         };
+#         if(is.logical(flat_params[[i]][[1]]) & flat_params[[i]][[1]] == TRUE) {
+#             return(paste0("--",i))
+#         };
+#         if(is.logical(flat_params[[i]][[1]]) & flat_params[[i]][[1]] == FALSE) {
+#             return("")
+#         } else {
+#             return(paste0("--",i," ",flat_params[[i]]))
+#         }
+#     }
+# 
+#     flatten <- function(x) {
+#         repeat {
+#             if(!any(vapply(x,is.list, logical(1)))) return(x)
+#             x <- Reduce(c, x)
+#         }
+#     }
+#     temp_params <- params
+# 
+#     # Convert different reductions into string with CL arguments
+#     reductions_params <- sapply(names(temp_params$reductions), function(reduction_name) {
+#         if (length(temp_params$reductions[[reduction_name]]) == 0) {
+#             tmp <- paste0("--", reduction_name)
+#         } else {
+#             tmp <- paste0("--", reduction_name, " ", temp_params$reductions[[reduction_name]][1])
+#         }
+#         temp_params$reductions[[reduction_name]][1] <<- NA
+#         tmp
+#     })
+# 
+#     # temp_params$reductions <- list()
+#     # Filter empty strings
+#     reductions_params <- Filter(reductions_params, f = function(x) nchar(x) > 0)
+#     reductions_string <-  paste0(reductions_params, collapse = " ")
+# 
+#     # Flatten reduction
+#     flat_params <- flatten(temp_params$reductions)
+#     # Convert reduction parameters list to "--arg _" list
+#     flat_reduction_params <- sapply(names(flat_params), FUN = params_to_strings)
+#     # Filter empty strings
+#     flat_reduction_params <- Filter(flat_reduction_params, f = function(x) nchar(x) > 0)
+#     # Create string "--passes 0 --bit_precision 18" for parser
+#     reduction_params_string <- paste0(flat_reduction_params, collapse = " ")
+# 
+#     temp_params$reductions <- list()
+# 
+#     #Set learning mode string argument
+#     algorithm_string <- switch (temp_params$algorithm,
+#                                 sgd = {tmp <- ""; tmp},
+#                                 bfgs = {tmp <- "--bfgs"; tmp},
+#                                 ftrl = {tmp <- "--ftrl"; tmp},
+#                                 ksvm = {tmp <- "--ksvm"; tmp}
+#     )
+#     # Disable cache here, because it's checked in vwtrain and vwtest
+#     if (temp_params$general_params$cache) {
+#         temp_params$general_params$cache <- NA
+#     }
+#     # Flatten list
+#     flat_params <- flatten(temp_params[-c(1)])
+#     # Convert parameters list to "--arg _" list
+#     flat_params <- sapply(names(flat_params), FUN = params_to_strings)
+#     # Filter empty strings
+#     flat_params <- Filter(flat_params, f = function(x) nchar(x) > 0)
+#     # Create string "--passes 0 --bit_precision 18" for parser
+#     parameters_string <- paste0(flat_params, collapse = " ")
+#     parameters_string <- paste(algorithm_string, parameters_string, reductions_string, reduction_params_string, sep = " ")
+# 
+#     return(parameters_string)
+# }
```

Comment:
The entire file seems to be empty / commented out.  Maybe omit this?

Reply:
Problem here, is that `roxygen2` can't see anything in this file when I use `R CMD check`.

Reply:
You must be confused over something else. There are no roxygen2 tags in the file!

Besides, roxygen2 is a _helper_. It should not govern what you include in a release.  This is supposed to be a code file, it has no code. 

What are we missing here?